docs/sync.js

```diff
@@ -34,6 +34,15 @@
  * @property {string} version - The version of the app.
  */
 
+/**
+ * This describes the options to configure client reset.
+ * @typedef {Object} Realm.App.Sync~ClientResetConfiguration
+ * @property {string} mode - Either "manual" (deprecated, see also `Realm.App.Sync.initiateClientReset()`) or "discardLocal" (download a fresh copy from the server).
+ * @property {callback(localRealm)|null} [clientResetBefore] - called before sync initiate client reset.
+ * @property {callback(localRealm, remoteRealm)|null} [clientResetAfter] - called after client reset has been executed.
```

Comment:
Up to you, but it might be more clear to use the terms `before` and `after`. Technically, these instances are both pointing to the same underlying Realm, just different versions of it.

Reply:
Good point